apps/vite/src/components/calendar/TimelineView.tsx

```
@@ -0,0 +1,125 @@
+        currentNepaliDate ||
+      (Number.parseInt(data.calendarInfo.dates.bs.day.en ?? "") >=
+        currentNepaliDate - 3 &&
+        Number.parseInt(data.calendarInfo.dates.bs.day.en ?? "") <=
+          currentNepaliDate + 3)
+    )
+  })
+
+  return (
+    <div className="flex flex-col divide-y divide-indigo-200">
+      {weekData.map((data) => (
+        <div className="h-[120px] flex items-center ">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)